azkaban-common/src/main/java/azkaban/utils/CaseInsensitiveConcurrentHashMap.java

```diff
@@ -0,0 +1,39 @@
+/*
+* Copyright 2018 LinkedIn Corp.
+*
+* Licensed under the Apache License, Version 2.0 (the “License”); you may not
+* use this file except in compliance with the License. You may obtain a copy of
+* the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an “AS IS” BASIS, WITHOUT
+* WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+* License for the specific language governing permissions and limitations under
+* the License.
+*/
+package azkaban.utils;
+
+import java.util.concurrent.ConcurrentHashMap;
+
+public class CaseInsensitiveConcurrentHashMap<V> extends ConcurrentHashMap<String, V> {
```

Comment:
Add a class doc?

e.g.
The name doesn't say if the map key is case insensitive or the value will also be case insensitive.

Reply:
Sure, added the class doc.